include/deal.II/differentiation/ad/adolc_math.h

```diff
@@ -66,11 +66,11 @@ namespace std
         return func_from(static_cast<const badouble &>(x),                     \
                          static_cast<const badouble &>(y));                    \
       }                                                                        \
-      inline adouble func_to(const double &x, const adouble &y)                \
+      inline adouble func_to(const double x, const adouble &y)                 \
```

Comment:
Extra white space 

Same below

Reply:
No, that is for aligning the "\" at the end of the line.

Reply:
Ah, right. I did not scroll far enough